### PR TITLE
Update docs for Wii compilation of RetroArch Wii

### PR DESCRIPTION
**Pull request by saulfabregwiivc**
Fix some bad spelling on the Wii docs for compiling, add some info, add another way for compiling cores ;)

### CODE REVIEW

File: docs/development/retroarch/compilation/wii.md [hunk 1]
```diff
@@ -18,18 +18,18 @@ For subsequent builds you only need to pull the changes from the repo
     cd retroarch
     git pull
 
-To update your local copy from the repository run git pull
+To update your local copy from the repository of RetroArch, run git pull
```
Comment (hizzlekizzle) [inline docs/development/retroarch/compilation/wii.md]: I agree that the original wording was ambiguous, but I think this should probably read: "To update your local copy of the RetroArch repository, run git pull".

File: docs/development/retroarch/compilation/wii.md [hunk 2]
```diff
@@ -18,18 +18,18 @@ For subsequent builds you only need to pull the changes from the repo
...
 
 ### Building RetroArch separately
 
 First, you need to compile [Salamander](../glossary.md#salamander).
-To compile Salamander (for Wii) run:
+For compile Salamander (for Wii) run:
```
Comment (hizzlekizzle) [inline docs/development/retroarch/compilation/wii.md]: I believe "To" is correct here.

File: docs/development/retroarch/compilation/wii.md [hunk 3]
```diff
@@ -18,18 +18,18 @@ For subsequent builds you only need to pull the changes from the repo
...
 
     make -f Makefile.wii.salamander
 
 Rename the file retroarch-salamander_wii.dol as boot.dol. This file is the frontend launcher for the other cores (indeed files containing core and frontend).
 
-Second, to compile RetroArch for Wii (the core and the frontend), rename the compiled core as 'libretro_wii.a' (see bellow how to compile a core), put it in the RetroArch directory and run:
+Second, to compile RetroArch for Wii (the core and the frontend), rename the compiled core as 'libretro_wii.a' (see below how to compile a core on "Building Cores" section), put it in the RetroArch directory and run:
```
Comment (hizzlekizzle) [inline docs/development/retroarch/compilation/wii.md]: Good catch!

I think the preposition "on" should be changed to "in the".

File: docs/development/retroarch/compilation/wii.md [hunk 4]
```diff
@@ -77,3 +77,22 @@ In case you only want to build one and/or more cores instead of all, you can spe
     ./libretro-build-wii.sh snes9x2010 fceumm
 
 Once finished, you can find the libretro cores inside directory `dist/wii`.
+
+Another way for compile the cores, is cloning the core's repository (for example, FCEUmm) from [GitHub](https://github.com/orgs/libretro/repositories).
```
Comment (hizzlekizzle) [inline docs/development/retroarch/compilation/wii.md]: "Another way _to compile cores is by_ cloning the core's repository..."

File: docs/development/retroarch/compilation/wii.md [hunk 5]
```diff
@@ -77,3 +77,22 @@ In case you only want to build one and/or more cores instead of all, you can spe
...
+In this case, for clone FCEUmm-Libretro repo:
```
Comment (hizzlekizzle) [inline docs/development/retroarch/compilation/wii.md]: for -> to

File: docs/development/retroarch/compilation/wii.md [hunk 6]
```diff
@@ -77,3 +77,22 @@ In case you only want to build one and/or more cores instead of all, you can spe
...
+    cd libretro-fceumm
+
+For subsequent builds you only need to pull the changes from the core's repo (ex., FCEUmm)
+
+    cd libretro-fceumm
+    git pull
+
+To update your local copy from the repository of the core (in this case FCEUmm), run git pull
+
+For compile the core for Wii (in this case, FCEUmm) run:
```
Comment (hizzlekizzle) [inline docs/development/retroarch/compilation/wii.md]: For -> To

**Comment by hizzlekizzle** [comment]
Thanks for taking the time to improve the doc!

I added a few notes.

**Comment by saulfabregwiivc** [comment]
> I added a few notes.

Sorry i'm not a GitHub expert but where I can see your notes?

**Review by hizzlekizzle — CHANGES_REQUESTED** [review]

**Comment by hizzlekizzle** [comment]
oh whoops, I think they were not visible to you. sorry about that. they should show up now.

**Comment by saulfabregwiivc** [comment]
Fixed the spelling issues, considering your suggestions.

**Pull request merged**